po/da.po

```diff
@@ -18673,9 +18665,8 @@ msgid ""
 msgstr ""
 
 #: src/exercises/concurrency/chat-app.md:127
-#, fuzzy
 msgid "Running the binaries"
-msgstr "Afvikling af kurset"
+msgstr "Kørsel af binære filer"
```

Comment:
Can we keep "Afvikling"? I much prefer that to "Kørsel".

Reply:
Yeha, certainly!

```suggestion
msgstr "Afvikling af binære filer"
```